custom_components/easee/const.py

```diff
@@ -412,7 +412,7 @@
     "reason_for_no_current": {
         "key": "state.reasonForNoCurrent",
         "attrs": [],
-        "units": "",
+        "units": None,
```

Comment:
I dont recall whether the the choice of an empty string was intentional or not. All state changes will be entered in the logbook if None is used.